engine/src/main/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActor.scala

```diff
@@ -108,7 +111,7 @@ class JobTokenDispenserActor(override val serviceRegistryActor: ActorRef,
 
     if (nextTokens.nonEmpty) {
       val hogGroupCounts = nextTokens.groupBy(t => t.queuePlaceholder.hogGroup).map { case (hogGroup, list) => s"$hogGroup: ${list.size}" }
-      log.info(s"Assigned new job $tokenTypeDescription tokens to the following groups: ${hogGroupCounts.mkString(", ")}")
+      log.info(s"Assigned new job $dispenserType tokens to the following groups: ${hogGroupCounts.mkString(", ")}")
```

Comment:
TOL: Putting a changeable value in the middle of this string might interrupt anything that's keying off the log string (probably only humans or bookmarked/documented search strings today). 

Maybe we could append rather than in-fix this information, eg:
```suggestion
      log.info(s"Assigned new job execution tokens to the following groups (for $dispenserType): ${hogGroupCounts.mkString(", ")}")
```

Reply:
hmm restart tokens are not execution tokens though